test/extended/builds/revision.go

```diff
@@ -25,8 +25,15 @@ var _ = g.Describe("[builds] build have source revision metadata", func() {
 
 	g.Describe("started build", func() {
 		g.It("should contain source revision information", func() {
-			g.By("starting the build with --wait flag")
-			err := oc.Run("start-build").Args("sample-build", "--wait").Execute()
+			g.By("starting the build with")
+			err := oc.Run("start-build").Args("sample-build").Execute()
```

Comment:
In general, in cases where the original test author did not care about the output of start-build, I did not introduce acquiring/dumping it into the test case (in the majority of cases, it is just the build name). No reason not the follow the pattern here if you prefer it.